### PR TITLE
docs: update contributing guidelines

### PR DESCRIPTION
**Pull request by mainawycliffe**
This an update to contributing guidelines to direct that patches and other small fixes be deployed to the master branch, while new features especially major features should be deployed to beta. This will allow us to deploy fixes as soon as they are approved and not make people wait for a major release for bug fixes. As is already happening today, the CI will keep both master and beta in sync, so any changes to master will be automatically merged to PR. 

This addresses some of the issues raised in this PR - #470

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -120,7 +128,7 @@ from the main (upstream) repository:
 - Check out the beta branch:
```
Comment (micimize) [inline CONTRIBUTING.md]: this is inconsistent now - also idk if we need a section for after the PR, but we might as well make it a script they could just execute

Comment (mainawycliffe) [reply CONTRIBUTING.md]: Yeah, the section might be unnecessary, we copied this I think from Angular contributing guidelines. Could do with a bit more polishing.

**Comment by mainawycliffe** [comment]
@HofmannZ can you approve this?

**Pull request merged**